metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppet-gitlab",
-  "version": "1.16.2-rc0",
+  "version": "2.0.0-rc0",
```

Comment:
Set this to 2.0.0. When you do the `bundle release` after merging, the Rake task should do a commit to bump the version to the next version-rc0 in master.